test/e2e/configs.go

```diff
@@ -207,35 +207,35 @@ groups:
   - name: example
     interval: 30s
     rules:
-      - record: job:http_inprogress_requests:sum
-        expr: sum by (job) (http_inprogress_requests)
+      - record: id_network_type
+        expr: 0 * topk by (ebs_account) (1, max by (ebs_account,account_type,internal,email_domain) (label_replace(label_replace(label_replace(subscription_labels{email_domain="domain1.com"}*0+5, "class", "Internal", "class", ".*") or label_replace(subscription_labels{class!="Customer",email_domain=~"(.*\\.|^)domain2.com"}*0+4, "class", "Internal", "class", ".*") or (subscription_labels{class="Customer"}*0+3) or (subscription_labels{class="Partner"}*0+2) or (subscription_labels{class="Evaluation"}*0+1) or label_replace(subscription_labels{class!~"Evaluation|Customer|Partner"}*0+0, "class", "", "class", ".*"), "account_type", "$1", "class", "(.+)"), "internal", "true", "email_domain", "domain1.com|(.*\\.|^)domain2.com") ))
```

Comment:
got it from [observatorium-metrics-template.yaml ](https://github.com/rhobs/configuration/blob/3978c8609271eb30fe16492af5f6b507ea86e139/resources/services/observatorium-metrics-template.yaml#L1340) in `rhobs/configuration`
I've just modified to be a more general expr, replacing `redhat` and `ibm` by `domain1` and `domain2`